src/scaffoldfitter/fitter.py

```diff
@@ -852,6 +853,37 @@ def getDataRMSAndMaximumProjectionErrorForGroup(self, groupName):
 
         return None, None
 
+    def getModelWorstElementJacobianInfo(self, mesh_group=None):
```

Comment:
getLowestElementJacobian(self, mesh_group=None)
Add comment that a value <= 0.0 is bad, either zero or negative volume / left-handed coordinates.